VRA/README-VRA.md

```diff
@@ -0,0 +1,77 @@
+
+## Integrating vRealize Orchestrator with VRA
+
+    
+    *  Open Cloud Assembly & Click on Orchestrator Control Center.
+   ![alt text](./images/cloud_assembly.png)
+   
+    *  On the Infrastructure tab from the left side scroll down to the Connection  Integration. 
+   ![alt text](./images/integration.png)
+    
+    *  Click on ADD INTEGRATION.
+   ![alt text](./images/new_integration.png)
+    
+    *  Select vRealize Orchestratoe from the options.
+   ![alt text](./images/select_vro.png)
+   
+    *  Provide the details of the Vro instance and click on ADD button.
```

Comment:
Fill in the details of the VRO instance required to be linked with VRA and click on ADD button.